packages/code/src/Syntax.tsx

```diff
@@ -12,36 +11,6 @@ import { injectGlobalStyles } from './globalStyles';
 import renderingPlugin, { TableContent } from './renderingPlugin';
 import { SyntaxContext } from './SyntaxContext';
 
-export function expandRangeTuple(tuple: [number, number]): Array<number> {
-  const [lower, upper] = [...tuple].map(bound => Math.max(bound, 0)).sort();
-  const maxHighlightingDifference = 499;
-
-  // Make sure passing large numbers doesn't freeze the browser
-  const clampedUpperBound = Math.min(lower + maxHighlightingDifference, upper);
-
-  const expandedRange = [];
-
-  for (let i = lower; i <= clampedUpperBound; i++) {
-    expandedRange.push(i);
-  }
-
-  return expandedRange;
-}
-
-export function normalizeLineHighlightingDefinition(
-  numbers: Array<number | [number, number]>,
-): Array<number> {
-  return flatten(
-    numbers.map(item => {
-      if (item instanceof Array) {
-        return expandRangeTuple(item);
-      }
-
-      return item;
-    }),
-  );
-}
-
 type FilteredSupportedLanguagesEnum = Omit<
   typeof SupportedLanguages,
   'Cs' | 'JS' | 'TS'
```

Comment:
Can we add a comment here that just explains that these are aliases/dupes

Reply:
Sure